pythran/config.py

```diff
@@ -223,6 +221,7 @@ def parse_define(define):
 
     # numpy specific
     if python:
+        import numpy
```

Comment:
why moving the import here?

Reply:
ah no need, `numpy` is a hard requirement. this is the only place it's needed, and I thought there was a non-python case to worry about. but there's not, I'll move it back